app-editors/qhexedit2/qhexedit2-0.8.4.ebuild

```diff
@@ -34,6 +34,8 @@ RDEPEND="
 "
 DEPEND="${RDEPEND}"
 
+S="${WORKDIR}/${PN}-ponyprog"
```

Comment:
Why are you changing the old ebuild?

Reply:
You still didn't answer this.

Reply:
was comment with rearrange and broken copyright. drop it here?